cuda_core/cuda/core/experimental/_launch_config.pyx

```diff
@@ -127,7 +137,42 @@ cdef class LaunchConfig:
                 f"block={self.block}, shmem_size={self.shmem_size}, "
                 f"cooperative_launch={self.cooperative_launch})")
 
+    cdef cydriver.CUlaunchConfig _to_native_launch_config(self):
+        _lazy_init()
+        # TODO: memset to zero?
+        cdef cydriver.CUlaunchConfig drv_cfg
```

Comment:
Do these types have a deterministic state after they've been created? Or does no ctor run leaving them in whatever state the bits on the stacks dictate?  If its the latter, I'd memset the structures to be zero.

Reply:
Yeah this is Cython saying "create this struct on the stack." I actually don't think we need to zero-initialize them. I read the API reference and don't find any requirement for these two structs, and in the end we overwrite all of their members so we are cool.